stories/exercises/06_exercise.mdx

```diff
@@ -0,0 +1,53 @@
+# 06. Customize components using settings.
+
+In the last exercise, we familiarized ourselves with the USWDS theme settings. In this section, we'll take that a step further by using component specific settings.
+
+Each USWDS component with settings will have this section. It is a snapshot of what customizations are possible without needing to write custom styles.
+
+**Tip:** The [Component settings](https://designsystem.digital.gov/documentation/settings/#component-settings-2) section of the Settings page provides a comprehensive list of all available component settings, their default values, and what type of token or value they take!
+
+1. Visit the [banner component](https://designsystem.digital.gov/components/banner/) page. Find the "Banner settings" section.
+
+1. To get started, lets try updating the font to a serif font. In `uswds-theme.scss`, add the following line to your settings map:
+    ```scss
+    $theme-banner-font-family: "serif",
+    ```
+1. Easy, right? Let's add a bit more color to our banner. Let's see what would happen if we brought back our accent color:
+    ```scss
+    $theme-banner-background-color: "accent-cool"
+    ```
+    The banner background color updated, but if you take a close at the banner link the contrast doesn't look great. If you take a look at the script running in the terminal window you'll notice a bunch of color contrast warnings have popped up.
+
+    USWDS does its best to ensure appropriate color contrast throughout the design system by setting good defaults. If a value is changed, it will check again where possible to ensure compliance. If the new color does not meet compliance standards, a fallback will be used.
+
+    If the automated checks determine neither color meets contrast requirements, we'll allow you to override but give you a warning. If we inspect this warning, we learn that the link color and it's fallback do not meet color contrast minimums.
```

Comment:
```suggestion
    If the automated checks determine neither color meets contrast requirements, we'll allow you to override with a contrast warning. If we inspect this warning, we learn that the link color and it's fallback do not meet color contrast minimums.
```

Reply:
Resolved in [30ff7e3](https://github.com/Bixal/storybook-rapid-prototyping/pull/19/commits/30ff7e3a62135fe81aea375663e1d8c45ab6c99c)